### ISSUE TITLE
You're forked ;-)

### ISSUE THREAD

**Issue by kuraga** [Raynos/vdom-thunk#15]
https://github.com/kuraga/vnode-immutable-thunk

**Comment by neonstalwart** [comment]
it's not a problem, but why?  what was missing/broken/... here?

**Comment by kuraga** [comment]
No, no. It's not an opposition, don't you you see smile in title? :smile: 

I'm working on https://github.com/kuraga/rotorjs-concept and there are some features I want to introduce which are doubtful for this package.

Now differences are custom args and renders equality functions.

P.S. It's just an advertise, feel free to close.

**Comment by neonstalwart** [comment]
i realize it's friendly but i was just curious what was not flexible enough or was missing from here for you.  i see now what some of the differences are.

btw, you don't include a way to provide a key.  you're missing a potentially significant optimization by not allowing for a key.

**Comment by kuraga** [comment]
Why not? Just use [`proto` argument](https://github.com/kuraga/vnode-immutable-thunk/blob/master/API.md#proto). I just unified this (but, maybe, in bad-way manner)

**Comment by neonstalwart** [comment]
i see.
